resources/views/layouts/navbar.blade.php

```diff
@@ -64,6 +65,24 @@
                 $newItem['isCustom'] = count($itemsInCustom) > 0;
                 $menuItems[] = $newItem;
             }
+            // New logic to highlight options in Top navbar when Analytics package is installed
+            $falseFlag = true;
+            foreach ($menuItems as $key => $menuItem) {
+                if ($menuItem['link'] === url()->current()) {
+                    if ($menuItem['isActive'] !== true) {
+                        $menuItems[$key]['isActive'] = false;
+                    } else {
+                        $falseFlag = false;
+                    }
+                } else {
+                    $menuItems[$key]['isActive'] = false;
+                }
+                // Admin link is different as url()->current() because controller route adds /users
+                if ($menuItems[$key]['title'] === 'Admin' && $falseFlag === true) {
```

Comment:
Is there a way to avoid the harcode value 'Admin'

Reply:
I'm open to hear another option

Reply:
An improvement was implemented to avoid Hardcode